Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -2230,7 +2232,14 @@ public struct Data : Equatable, Hashable, RandomAccessCollection, MutableCollect
         }
     }
 
-    @available(FoundationSpan 6.2, *)
```

Comment:
Since we don't need these availability annotations anymore, can we remove them from the package manifest / CMake files?

Reply:
I forgot that part. Will do!